About.html

```diff
@@ -28,41 +28,49 @@
   </header>
 
   <header class="navDesktop">
-    <nav class="navbar bg-dark">
-      <div class="container-fluid">
+    <div>
```

Comment:
## [OPTIONAL]
Super cool navbar layout. Well done 👍  butI think you missed the little social media navigation bar at the top. 😀

Let's add it 👇

![](https://i.imgur.com/ns8Fdkk.png)